src/index.ts

```diff
@@ -143,6 +144,14 @@ class Radar {
     }
   }
 
+  public static trackVerified(params: RadarTrackParams = {}) {
+    try {
+      return VerifyAPI.trackVerified(params);
+    } finally {
+      ConfigAPI.getConfig(params); // call with updated permissions
```

Comment:
It doesn't seem like trackVerified is making any calls that would modify permissions

Reply:
Agreed, will simplify

Reply:
Done